events.hpp

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+#include <iostream>
```

Comment:
Ten include jest tu niepotrzebny.

Reply:
Bez niego nie działa :d
Ma być w events.cpp?

Reply:
Ma być tam, gdzie używana jest zawartość iostream - a w tym nagłówku nie widzę żadnych jego użyć.

(Tak - jeśli w `events.cpp` używasz iostream, to tam włącz ten nagłówek.)